### DISCUSSION TITLE
blog/deriving-ols-estimator

### DISCUSSION THREAD

**Discussion by giscus** [Samdasoo1076/blog#10] [comment]
# blog/deriving-ols-estimator

수학 조판을 만들어 마크다운 문법에서 수학적 노트 만들기

https://blog-samdasoo.vercel.app/blog/deriving-ols-estimator

**Comment by levocation** [comment]
테트레이션의 원형을 표현할려면 어떻게 해야 하나요?

**Reply by Samdasoo1076** [reply]
# 테트레이션(Tetration)은 주어진 수 \(a\)와 정수 \(n\)에 대해 \(a\)를 \(n\)번 거듭제곱하는 것입니다.

$$
^n a = \underbrace{a^{a^{a^{\cdot^{\cdot^{\cdot^a}}}}}}_{n \text{ times}}
$$

## 예를 들어, 3을 4번 테트레이션하면:

$$
^4 3 = 3^{3^{3^3}}
$$

## 위 식을 순서대로 계산하면 다음과 같습니다:

$$
3^3 = 27
$$

$$
3^{27} = 7625597484987
$$

## 따라서:

$$
^2 3 = 3^3 = 27
$$

$$
^3 3 = 3^{3^3} = 3^{27} = 7625597484987
$$

입니다. 감사합니다.
